src/main/java/com/textmessenger/service/CommentServiceImpl.java

```diff
@@ -14,16 +15,19 @@
 public class CommentServiceImpl implements CommentService {
 
   private final CommentRepository commentRepository;
+  private final NotificationService notificationService;
 
-  CommentServiceImpl(CommentRepository commentRepository) {
+  CommentServiceImpl(CommentRepository commentRepository, NotificationService notificationService) {
     this.commentRepository = commentRepository;
+    this.notificationService = notificationService;
   }
 
   @Override
   public void createComment(Post post, User user, Comment comment) {
     comment.setPost(post);
     comment.setCommentator(user);
-    commentRepository.save(comment);
+    Comment save = commentRepository.save(comment);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this useless assignment to local variable "save". ([squid:S1854](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1854))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=139&issues=AWV3RcoaelKyZq-pEF1X&open=AWV3RcoaelKyZq-pEF1X)

[ISSUE_KEY]: AWV3RcoaelKyZq-pEF1X